extensions/2.0/Vendor/EXT_feature_metadata/schema/featureClass.property.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property class",
+  "type": "object",
+  "description": "A property class.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "name": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The name of the property, e.g. for display purposes. The string is UTF-8 encoded."
+      },
+      "semantic": {
+          "type": "string",
+          "minLength": 1,
+          "description": "Identifies a property with a well-known meaning. The string is UTF-8 encoded. Semantics defined by this extension include `NAME` and `ID`"
+      },
+      "description": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The description of the property. The string is UTF-8 encoded."
+      },
+      "elementType": {
+          "type": "string",
+          "enum": [
+              "BOOLEAN",
+              "INT8",
+              "UINT8",
+              "INT16",
+              "UINT16",
+              "INT32",
+              "UINT32",
+              "INT64",
+              "UINT64",
+              "FLOAT16",
+              "FLOAT32",
+              "FLOAT64",
+              "BLOB",
+              "STRING",
+              "ARRAY",
+              "VARIABLE_SIZE_ARRAY"
+          ],
+          "description": "The type of each element of the property array. An `elementType` is either a basic type used for `componentType`, an `ARRAY`, or a `VARIABLE_SIZE_ARRAY`. If `ARRAY` or `VARIABLE_SIZE_ARRAY` is used, then `componentType` must also be specified. `ARRAY` is a fixed-length array, and requires a `componentsPerElement` property. `VARIABLE_SIZE_ARRAY`, `BLOB` and `STRING` are variable-length unless the feature table has `elementByteLength` defined, in which case values must be exactly this length. When `featureTable.elementByteLength` is not specified, `VARIABLE_SIZE_ARRAY`, `STRING`, and `BLOB` also require a corresponding `offsetBufferView`. This is used to indicate where each element starts in the buffer. See further notes about offset buffers in the description for `componentType`."
+      },
+      "componentType": {
+          "enum": [
+              "BOOLEAN",
```

Comment:
Also for the bit vector, how is this laid out in memory? Is the goal to make 

`boolean_property[i] === (bignum(boolean_property) >> i) & 1`? (i.e. bits are numbered from right to left in binary)

And either way, what endianness are we using here?

Reply:
Good question about bit layout. We should explicitly say little endian since that's what 3D Tiles and glTF already do.

Reply:
how much does LE help here?

Suppose we have a bunch of bytes, I'll label them with letters:

```
ABCDEFGHIJKLMN

if the bytes were read in a 32-bit machine, you'd probably store it:

DCBA HGFE LKJI xxNM

If the bytes are read in with 64-bit machine, it would be best to order them

HGFEDCBA xxNMLKJI

Or we can interpret LE as "put everything backwards"

xxNMLKJIHGFEDCBA

but now if you want to know where the first bit of A is, you need
to know the total byte length to find it.
```